app.py

```diff
@@ -10,38 +11,39 @@
 @app.route("/home")
 def home_page():
 
-    # QUERY CALL ( Get ALL location id and names for display )
+    # Query call to get all locations
     locations = sql_queries.get_locations()
 
     return render_template('index.html', locations=locations)
 
 
+# Route takes in a location id string and may take in a rack id string.
+# If no rack id is provided, it will have a default value of None
 @app.route('/location/<string:location_id>/', defaults={'rack_id': None},
            methods=['POST', 'GET'])
 @app.route('/location/<string:location_id>/<string:rack_id>',
            methods=['POST', 'GET'])
 def location_page(location_id, rack_id):
 
-    # QUERY CALL ( Get the dimensions for the table )
+    # Query to get the dimensions for the table
     data = sql_queries.get_table_size(location_id)
 
+    if("ST" in location_id):
+        cells_per_bay = 2
+    elif("DIS" in location_id):
+        cells_per_bay = 3
+    else:
+        cells_per_bay = 1
+    tableType = check_table_type(location_id)
```

Comment:
Try and think of a way of storing this information somewhere that isn't hard coded. As I said yesterday, we don't want to hard code values for distinct location ids in the code